.github/workflows/build.yaml

```diff
@@ -94,3 +94,50 @@ jobs:
           echo "::group::Log from these tests"
           [ ! -f Testing/Temporary/LastTest.log ] || cat Testing/Temporary/LastTest.log
           echo "::endgroup::"
+
+
+  windows-mingw:
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: msys2 {0}
+    
+    steps:
+      - uses: actions/checkout@v4
+      
+      - uses: msys2/setup-msys2@v2
+        with:
+          msystem: MINGW64
+          update: true
+          install: >-
+            git
+            mingw-w64-x86_64-gcc
+            mingw-w64-x86_64-gcc-fortran
+            mingw-w64-x86_64-cmake
+            mingw-w64-x86_64-openblas
+            mingw-w64-x86_64-parmetis
+            make
```

Comment:
It'd probably be a good idea to install the `base-devel` group instead of only `make`.

Reply:
@mmuetzel thanks for the comment. Do you want to share a patch or send a PR to this specific branch? I think it should be possible to give you write access to this specific branch. But I rather core devs to do that.

Reply:
Why can't you make that change yourself? It shouldn't be hard.

Reply:
@mmuetzel fair point. But I did not explain myself properly. Given your experience it would be great if we could give you write access on some of the MSYS2 / Mingw-w64 / Windows-related branches. Of course, if you are interested.